.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Question, does this have to be in the package?

Reply:
It doesn't have to be in this pull request. @ceiroa suggested I commit it with my changes so we don't have to manually remove the .idea folder every time.